crates/matrix-sdk-ui/src/timeline/mod.rs

```diff
@@ -414,6 +416,68 @@ impl Timeline {
         Ok(())
     }
 
+    /// Send a reply to the given event.
+    ///
+    /// Currently only supports `m.room.message` events whose event ID is known
+    /// and will log an error and do nothing for other events. Please check
```

Comment:
*narrator's voice* It did not, in fact, log an error, but instead would return an `Err` result.

(Can you update the comment please?)